<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/icons/main.ico</normaloff>
    <normalon>:/icons/icons/main.ico</normalon>:/icons/icons/main.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="BQGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>541</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>几何</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>82</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="circleBtn">
         <property name="text">
          <string>圆</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ellipseBtn">
         <property name="text">
          <string>椭圆</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="squareBtn">
         <property name="text">
          <string>正方形</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rectangleBtn">
         <property name="text">
          <string>矩形</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearBtn">
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveBtn">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadBtn">
         <property name="text">
          <string>加载</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>网格</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab3">
     <attribute name="title">
      <string>材料</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab4">
     <attribute name="title">
      <string>计算</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BQGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">bqgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
